app/src/main/java/com/elmeradrianv/shesafe/LoginActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.elmeradrianv.shesafe;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+
+
+import com.parse.ParseUser;
+
+public class LoginActivity extends AppCompatActivity {
+    public static final String TAG="LoginActivity";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        EditText etUsername;
```

Comment:
As discussed on VC let's move these to be inline below:

EditText etUsername = findViewById(R.id.etUsername);